packages/walletconnect/src/utils.ts

```diff
@@ -66,3 +81,18 @@ export async function getBestUrl(urls: string[], timeout: number): Promise<strin
     })
   })
 }
+
+/**
+ * @param chains - An array of chain IDs
+ * @param defaultChainId - An ID to move to the front of the array (first element is always the default chain)
+ */
+export function orderToSetDefaultChain(chains: number[], defaultChainId: number) {
```

Comment:
This function name is unclear to me, do you think something like `getChainsWithDefault(chain: number[], defaultChain: number)` would clarify it?

Reply:
We actually spent some time with @cmcewen looking to find the best name and this was our best case. Personally, both options (this one and your suggestion) have their pros and cons. 

My yet another alternative is:
- `setDefaultChain(chains, defaultChain)`, which is more explicit, but takes away `order` from the name, making it less explicit about what it actually does.

What do you think?

Reply:
I think the intent is more important than the mechanism, so I don't mind `order` being omitted. `setDefaultChain` could be interpreted to be modifying `chains` in place. `orderToSetDefaultChain` doesn't read well, so I don't know what it's doing.

I'd still prefer `getChainsWithDefault` to make it clear that you're making a copy of `chains` with the `defaultChain` in the correct place.

Reply:
Ok!